crates/file/src/file_list.rs

```diff
@@ -0,0 +1,23 @@
+use crate::blob::File;
+
+pub struct FileList {
+    inner: Vec<File>,
+}
+
+impl std::convert::Into<FileList> for web_sys::FileList {
```

Comment:
We should implement `From<web_sys::FileList> for FileList` and then we get the `Into` impl for free as well because there is a blanket `impl<T, U> Into<T> for U where T: From<U>`.

Reply:
Same for all the other `Into` impls. Generally, you never want to implement `Into` manually, only `From`.

Also, `Into` and `From` are in the prelude, so you don't need to fully qualify them.

Reply:
I always get this backwards. Will make the change.